docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,75 +1,298 @@
-[[get-started-docker]]
-== Running the {stack} on Docker
+[[get-started-stack-docker]]
+== Running the {stack} ("ELK") on Docker
+++++
+<titleabbrev>Running the {stack} on Docker</titleabbrev>
+++++
 
 The Elastic Docker registry contains Docker images for all the products in
 the {stack}: https://www.docker.elastic.co/.
 
-[float]
-[[run-stack-docker]]
-=== Run with Docker Compose
+You can <<run-docker-secure,start the {stack} in Docker>> with security enabled
+and configured by default. This option is great for quickly getting started with
+{es} and {kib} for testing and development.
 
-To get the default distributions of {es} and {kib} up and running in Docker,
-you can use Docker Compose.
+You can also <<get-started-docker-tls,start the {stack} with Docker Compose>> to
+create a secured, multi-node cluster with a connected {kib} instance. This
+option requires more manual steps, but results in a more resilient cluster with
```

Comment:
we're trying to make it a single step too, so maybe we don't need defensive language here anymore ?

Reply:
I'll update this language in the subsequent PR that introduces the simplified Docker Compose configuration.